app/routes.test.js

```diff
@@ -216,17 +216,48 @@ describe('routes', () => {
   });
 
   describe('POST /solutions/capabilities-selector', () => {
-    it('should return the correct status and text if there is no error', () => {
+    it('should return the correct status and text if there is no error and one capability selected', () => {
+      solutionListPageContext.getSolutionsForSelectedCapabilities = jest.fn()
+        .mockImplementation(() => Promise.resolve(mockFilteredSolutions));
+      const app = new App().createApp();
+      app.use('/', routes);
+      return request(app)
+        .post('/solutions/capabilities-selector')
+        .send({ capabilities: 'C1' })
+        .expect(302)
+        .then((res) => {
+          expect(res.redirect).toEqual(true);
+          expect(res.text).toEqual('Found. Redirecting to /solutions/capabilities-selector.C1');
+        });
+    });
+
+    it('should return the correct status and text if there is no error and many capabilities selected', () => {
+      solutionListPageContext.getSolutionsForSelectedCapabilities = jest.fn()
+        .mockImplementation(() => Promise.resolve(mockFilteredSolutions));
+      const app = new App().createApp();
+      app.use('/', routes);
+      return request(app)
+        .post('/solutions/capabilities-selector')
+        .send({ capabilities: 'C1 C2 C3' })
+        .expect(302)
+        .then((res) => {
+          expect(res.redirect).toEqual(true);
+          expect(res.text).toEqual('Found. Redirecting to /solutions/capabilities-selector.C1%20C2%20C3');
```

Comment:
should these be separated by `+`?

Reply:
this is uri encoded - there was a bit in that article Sam posted about url encoding so this might change